### PR TITLE
DDF-5349 added Configure SSO instructions to admin docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates installation instructions to include SSO configuration

#### Select relevant component teams: 
@codice/docs 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@blen-desta
@brjeter
@clockard
@garrettfreibott
@lessarderic
@ricklarsen - Documentation

#### How should this be tested?
verify content

#### What are the relevant tickets?
Fixes: #5348 

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
+:title: System Configuration Settings
+:type: installing
+:status: published
+:summary: Configure Single Sign On(SSO) page of installer.
+:project: ${branding}
+:order: 07
+
+.[[_configuring_sso]]Configure Single Sign On(SSO)
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: ❓ There should be a space before `(SSO)`, right?

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+****
+Configure Single Sign On method: SAML or OIDC.
+
+.SAML SSO
+Enter the URLs for the IdP metatdata and set other options.
+
+IdP Server::
+Configure ${BRANDING}'s' internal Identity Provider Server.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: ✏️ Looks like a stray single quote there after the `s`.

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+. *_SP Metadata_*: The metatada of the desired Service Provider to connect to the internal IdP. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(`&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;`).
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: ✏️ Another case where there should be a space before the opening paren after `block`, I think?

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+. *_Require Signed AuthnRequests_*:
+Toggle whether or not to require signed AuthnRequests. When off, unsigned AuthnRequests will be rejected.
+
+. *_Limit RelayStates to 80 Bytes_*: Toggle whether or not to restrict the RelayState length to 80 bytes. When off, messages with RelayStates over 80 bytes will be rejected.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: ❓ Should we include a comment here about why we might want to reject `RelayState`s of over 80 bytes in length?

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+. *_Cookie Expiration Time (minutes)_*:
+Sets the cookie expiration time for Single Sign On, which determines how long the internal IdP will cache SAML assertions for later use. This value should match the lifetime of SAML assertions.
+
+IdP Client::
+Configure handling of incoming requests where SAML authentication is enabled.
+
+. *_IdP Metadata_*:
+The metadata of the desired IdP to authenticate with. Can be configured as an HTTPS URL (https://), file URL (file:), or an XML block(&lt;md:EntityDescriptor&gt;...&lt;/md:EntityDescriptor&gt;).
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: ✏️ Same comment about the space after `block`.

Comment (coyotesqrl) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: Also, should that block of code be inside a code fence? It is above.

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 2 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 3 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 4 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: Yes. I'll track that down

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 5 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: fixed

File: distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc (hunk 4 above)
Comment (coyotesqrl) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: This is in section 3.4.3 of [Bindings for the OASIS Security Assertion Markup Language (SAML) V2.0 – Errata Composite](https://www.oasis-open.org/committees/download.php/56780/sstc-saml-bindings-errata-2.0-wd-06-diff.pdf) 

> RelayState data MAY be included with a SAML protocol message transmitted with this binding. The value MUST NOT exceed 80 bytes in length and SHOULD be integrity protected by the entity creating the message.

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_managing/_installing/configure-single-sign-on.adoc]: updated

**Comment by ricklarsen** [comment]
build now

**Pull request merged**